### DISCUSSION TITLE
vote: migrate spark-style-guide to mrpowers-io

### DISCUSSION THREAD

**Discussion by MrPowers** [mrpowers-io/community#13] [Ideas]
project: https://github.com/MrPowers/spark-style-guide

purpose: we should update the Scala Spark & PySpark guides with more details.

It would also be cool to add flake8 linters to this repo to enforce these best practices!

**Comment by MrPowers** [comment]
+1

**Comment by SemyonSinchenko** [comment]
+1 (binding)

**Comment by zeotuan** [comment]
+1

**Comment by alfonsorr** [comment]
+1

**Comment by MrPowers** [comment]
Vote passed!

Repo migrated: https://github.com/mrpowers-io/spark-style-guide
